lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_builtin_method.rb

```diff
@@ -142,8 +142,8 @@ def self.miq_src_vm_destroy_all_snapshots(obj, _inputs)
       event_object_from_workspace(obj).src_vm_destroy_all_snapshots
     end
 
-    def self.miq_src_vm_disconnect_storage(obj, _inputs)
-      event_object_from_workspace(obj).src_vm_disconnect_storage
+    def self.miq_src_vm_disconnect_storage(_obj, _inputs)
+      # Method has been deprecated and will be removed in a future version
```

Comment:
I don't think we are using the word "deprecated" correctly here.  Would be more accurate to say that the `src_vm_disconnect_storage` logic has been removed and this method remains for backwards compatibility with older Automate models and/or users overriding `DestroyVM_Task_Complete` and `USER_REMOVE_VM_FINISHED` event instances.  See https://github.com/ManageIQ/manageiq-content/pull/472

The "will be removed in a future version" is right on.  👍

Reply:
That's a good point, updated